<html>
	<head>
		<title>
			Officers
		</title>
		<style type="text/css">
			<!--
				ul {list-style-type : none; font-size : large}
				.title {font-weight : bold}
 table.MsoNormalTable
	{mso-style-parent:"";
	font-size:10.0pt;
	font-family:"Times New Roman"}
 p.MsoNormal
	{mso-style-parent:"";
	margin-bottom:.0001pt;
	font-size:12.0pt;
	font-family:"Times New Roman";
	margin-left:0in; margin-right:0in; margin-top:0in}
			-->
		</style>
	</head>
	<BODY><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4> </FONT><DIV class=moz-text-plain lang=x-western style="FONT-SIZE: 13px; FONT-FAMILY: -moz-fixed" graphical-quote="true" wrap="true"><PRE wrap=""><FONT face="Times New Roman" size=4><STRONG>The UW Chapter of the Association for Computing Machinery's Committee on
Women (WACM) presents the second seminar in the series:

         Distinguished Lectures in Computational Biology

                          * * * * * *

                     Prof. Martha L. Bulyk</STRONG>

  Division of Genetics, Department of Medicine; Department of Pathology
      Harvard-MIT Division of Health Sciences &amp; Technology
      Brigham &amp; Women's Hospital and Harvard Medical School
      </FONT><A class=moz-txt-link-freetext href="http://the_brain.bwh.harvard.edu/contact_information.html"><FONT face="Times New Roman" size=4>http://the_brain.bwh.harvard.edu/contact_information.html</FONT></A>

<FONT face="Times New Roman" size=4><STRONG>                          10:30 am
                      Tuesday, April 4th
     Biotech Center Auditorium (1111 Biotech Center)
                       425 Henry Mall


"Genomic Analyses of Transcription Factors and Cis Regulatory Elements"

Abstract:

</STRONG>The interactions between transcription factors (TFs) and their DNA binding
sites are an integral part of the cellular regulatory networks that
control gene expression. Previously, we developed a DNA microarray-based
in vitro technology that allows the rapid, high-throughput
characterization of the DNA binding site sequence specificities of TFs in
a single day. We plan to use PBM data on the DNA binding specificities of
metazoan TFs for more accurate prediction of cis regulatory modules within
the vast noncoding portions of those organisms' genomes. Specifically, we
are inferring co-regulation by sets of TFs through an integrated analysis
of gene expression data, TF binding site motif data, and prediction of cis
regulatory modules. Examples of the prediction of cis regulatory modules
conforming to a core cis regulatory code inferred in such a way will be
shown for the founder cell population in the Drosophila embryonic
mesoderm.

<STRONG>            *********************

</STRONG>Co-hosted by the Genome Center of Wisconsin (GCW), and co-sponsored by the
Computation and Informatics in Biology and Medicine Training Program
(CIBM) and Women in Science and Engineering Leadership Institute (WISELI),
this series features research presentations by three highly regarded
research scholars.

This seminar by Prof Martha Bulyk is the second seminar in the.
Distinguished Lectures in Computational Biology

Each invited scholar will present a colloquium, targeted to a general
Computational Biology audience, and head an informal roundtable
discussion/Q&amp;A session with women students.

The first speaker of the series was Prof. Daphne Koller.
</FONT></PRE></DIV></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4><B>*****************************************************************************************</B></FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px">&nbsp;</PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman"><FONT size=4><B>UW Chapter, Association for Computing Machinery's Committee on Women (WACM) presents:</B>

<B>Distinguished Lectures in Computational Biology </B></FONT></FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>Co-sponsored by the Department of Biostatistics and Medical Informatics (</FONT><A href="http://www.biostat.wisc.edu"><FONT face="Times New Roman" size=4>http://www.biostat.wisc.edu</FONT></A><FONT face="Times New Roman" size=4>),
Computation and Informatics in Biology and Medicine Training Program (</FONT><A href="http://cibm.wisc.edu/"><FONT face="Times New Roman" size=4>http://cibm.wisc.edu/</FONT></A><FONT face="Times New Roman" size=4>) and </FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>WISELI (</FONT><A href="http://wiseli.engr.wisc.edu/"><FONT face="Times New Roman" size=4>http://wiseli.engr.wisc.edu/</FONT></A><FONT face="Times New Roman" size=4>) this series features research presentations by three highly regarded </FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>research scholars.</FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>&nbsp;</FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>Each invited scholar will present a colloquium, targeted to a general Computational Biology audience, </FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>and head an informal roundtable discussion/Q&amp;A session with female students.
</FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>The first of the three speakers is:</FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>&nbsp;</FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><B><FONT face="Times New Roman" size=4>Prof. Daphne Koller  (</FONT><A class=moz-txt-link-freetext href="http://ai.stanford.edu/~koller/bio.html"><FONT face="Times New Roman" size=4>http://ai.stanford.edu/~koller/bio.html</FONT></A><FONT face="Times New Roman" size=4>)
</FONT></B><FONT face="Times New Roman"><FONT size=4>Stanford University
<B><I><BLINK>Friday, October 28</BLINK>  at the Pyle Center (note unusual location)</I></B></FONT></FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>check listing for CIBM Retreat or WISELI meeting to locate rooms</FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px">
<FONT face="Times New Roman"><FONT size=4>10:45am - 11:45am       Women's roundtable discussion with Daphne Koller<SPAN class=moz-txt-citetags> </SPAN></FONT></FONT></PRE><PRE style="MARGIN-TOP: 0px; MARGIN-BOTTOM: 0px"><FONT face="Times New Roman" size=4>12:00pm - 1:15pm         Lunch
1:15 pm - 2:15pm          Colloquium talk on "Modeling Conservation and<SPAN class=moz-txt-citetags> </SPAN>Variation in Regulatory Networks"

</FONT><FONT face="Times New Roman"><FONT size=4><SPAN class=moz-txt-citetags> </SPAN>Abstract:
<SPAN class=moz-txt-citetags> </SPAN>      A regulatory module can be defined as a set of genes that is
controlled by the same regulation program, defined by the activity level
of a set of joint regulators. In [Segal et al., Nature Genetics 2003], we
described a genome-wide computational model for regulatory modules, and an
automated procedure for identifying such modules from gene expression
data. In this work, we study both the evolutionary conservation and the
evolutionary variation of regulatory modules. In one project*, we study
gene regulatory modules that have been conserved over evolution. By using
evolution as a filter, we derive biological signals that are much clearer
than in a single species, and also obtain insights into evolutionary
forces involving regulation. We describe a computational method for
discovering regulatory modules from multi-species gene expression data,
where a regulatory module is a set of co-regulated genes and the proteins
that regulate their activity. Our method explicitly biases the module
construction in favor of conserved function and conserved regulation. We
applied our method to a joint microarray data set measuring expression in
healthy and tumor brain tissue in human and mouse. Our analysis shows that
the resulting multi-species modules are significantly more predictive on
unseen expression measurements of human samples compared to modules
learned from human data alone. The resulting modules suggest novel
hypotheses relating to regulation of medulloblastoma.
In another project we study the effect of genetic sequence
variability within a population of individuals on regulatory modules. We
extend our model of regulation, allowing both sequence variation and
regulator expression to play a role in regulating the expression level of
target genes. Our method automatically constructs fine-grained mechanistic
explanations for the effect of sequence polymorphisms on regulatory
interactions. In particular, unlike standard approaches, it allows us to
distinguish between direct and indirect effects of genetic variations on
gene expression. We applied our method to a data set of yeast recombinants
generated by a cross between laboratory (BY) and wild (RM) strains of S.
cerevisiae (Brem &amp; Kruglyak, 2005), and show that it out-performs standard
approaches at explaining the expression data. Our analysis provides
several intriguing hypotheses regarding the effect of sequence
polymorphisms on gene regulation, including hypotheses involving
regulation via mRNA degradation and chromatin remodeling. One of these
hypotheses has recently been tested and validated in wet-lab experiments.
The analysis also suggests some interesting systems-level insights on
genetic variation and its effect on regulatory networks.</FONT></FONT></PRE>
	</BODY>
</html>